10/entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+if [ ! -z "$2" ]
+then
+  echo "Changing dir to $2"
+  cd $2
+fi
+
+echo "Using $1 to install dependencies"
+$1 install
```

Comment:
you said above `JS projet without dev dependencies` yet here you install dev deps no ?

Reply:
No, I install all the dependencies because I need a vue-cli to be able to launch the linter.

Reply:
ye ok then you need to fix the description

Reply:
yes already fixed